rolling/distribution.yaml

```diff
@@ -7181,6 +7181,21 @@ repositories:
       url: https://github.com/septentrio-gnss/septentrio_gnss_driver.git
       version: master
     status: maintained
+  sick_safetyscanners2_interfaces:
+    doc:
+      type: git
+      url: https://github.com/SICKAG/sick_safetyscanners2_interfaces.git
+      version: master
+    release:
+      tags:
+        release: release/rolling/{package}/{version}
+      url: https://github.com/SICKAG/sick_safetyscanners2_interfaces-release.git
```

Comment:
With our current rules, all releases into Rolling must use the `ros2-gbp` organization for the release repositories; see https://github.com/ros/rosdistro/blob/09be553dc16a84478274693fff2b11ab524007dd/REVIEW_GUIDELINES.md#L47 .